docs/swagger_v3/transactions.spec.yaml

```
@@ -1,4 +1,170 @@
+                type: string
+              contract_id:
+                $ref: '#/components/schemas/ContractAddress'
+              gas_used:
+                type: integer
+              return_type:
+                type: string
+        - title: GAMetaTx
+          allOf:
+          - $ref: '#/components/schemas/GAMetaTx'
+          - type: object
+            required:
+              - gas_used
+              - return_type
+            properties:
+              gas_used:
+                type: integer
+              return_type:
+                type: string
+        - title: NameClaimTx
+          allOf:
+          - $ref: '#/components/schemas/NameClaimTx'
+          - type: object
+            required:
+              - name_id
+            properties:
+              name_id:
+                $ref: '#/components/schemas/NameHash'
+        - $ref: '#/components/schemas/NamePreclaimTx'
+        - title: NameRevokeTx
+          allOf:
+          - $ref: '#/components/schemas/NameRevokeTx'
+          - $ref: '#/components/schemas/MdwName'
+        - title: NameTransferTx
+          allOf:
+          - $ref: '#/components/schemas/NameTransferTx'
+          - $ref: '#/components/schemas/MdwName'
+        - title: NameUpdateTx
+          allOf:
+          - $ref: '#/components/schemas/NameUpdateTx'
+          - $ref: '#/components/schemas/MdwName'
+        - $ref: '#/components/schemas/OracleExtendTx'
+        - title: OracleQueryTx
+          allOf:
+          - $ref: '#/components/schemas/OracleQueryTx'
+          - type: object
+            required:
+              - query_id
+            properties:
+              query_id:
+                $ref: '#/components/schemas/OracleQueryId'
+        - title: OracleRegisterTx
+          allOf:
+          - $ref: '#/components/schemas/OracleRegisterTx'
+          - type: object
+            required:
+              - oracle_id
+            properties:
+              oracle_id:
+                $ref: '#/components/schemas/OracleAddress'
+        - $ref: '#/components/schemas/OracleRespondTx'
+        - $ref: '#/components/schemas/PayingForTx'
+        - $ref: '#/components/schemas/SpendTx'
+    discriminator:
+      propertyName: type
+      mapping:
+        channel_close_mutual: '#/components/schemas/ChannelCloseMutualTx'
+        channel_close_solo: '#/components/schemas/ChannelCloseSoloTx'
+        channel_create: '#/components/schemas/ChannelCreateTx'
+        channel_deposit: '#/components/schemas/ChannelDepositTx'
+        channel_force_progress: '#/components/schemas/ChannelForceProgressTx'
+        channel_settle: '#/components/schemas/ChannelSettleTx'
+        channel_slash: '#/components/schemas/ChannelSlashTx'
+        channel_snapshot_solo: '#/components/schemas/ChannelSnapshotSoloTx'
+        channel_withdraw: '#/components/schemas/ChannelWithdrawTx'
+        contract_call: '#/components/schemas/ContractCallTx'
+        contract_create: '#/components/schemas/ContractCreateTx'
```

in the example from the issue
https://mainnet.aeternity.io/mdw/v3/transactions/th_2ZtAgxftmG8ts2VoxxPiCWr1q83gxHHmxE3kUsJQgnEgLwYxdV
```
    "type": "ContractCreateTx",
```
so, as I understand, here should be
```suggestion
        ContractCreateTx: '#/components/schemas/ContractCreateTx'
```